app/domain/order/report.rb

```diff
@@ -161,11 +161,14 @@ def filter_by_target(orders)
     end
 
     def filter_by_closed(orders)
-      return orders if params[:closed].blank?
-
-      orders
-        .where(order_statuses: { closed: true })
-        .where(period.where_condition('closed_at'))
+      return orders if params[:status_preselection].blank?
+      if params[:status_preselection] == 'closed'
+        return orders.where(order_statuses: { closed: true })
+                     .where(period.where_condition('closed_at')) 
+      end
+      return unless params[:status_preselection] == 'in_progress'
+        orders.where(order_statuses: { closed: false })
+      
```

Comment:
This needs at least reformatting, let's see... If I reformat, I notice that we are checking the same param over and over, so I switched to a case (pun intended)

```suggestion
      case params[:status_preselection]
      when nil, ""
        orders
      when 'closed'
        orders.where(order_statuses: { closed: true })
              .where(period.where_condition('closed_at')) 
      when 'in_progress'
        orders.where(order_statuses: { closed: false })
      end
```

This also nicely obsoletes the mixture of if and unless which made this an excellent cylon detector.

The first case could be a guard statement as well. My proposal is to keep this all at the same level.